pages/packages/private_storage.md

```diff
@@ -0,0 +1,54 @@
+# Link private storage
+
+This page provides details on how to link your private Amazon Web Services (AWS) Simple Storage Service (S3) storage to Buildkite Packages within your Buildkite organization. This process can only be conducted by [Buildkite organization administrators](/docs/packages/permissions#manage-teams-and-permissions-organization-level-permissions).
+
+By default, Buildkite Packages provides its own storage to house any packages, container images and modules stored in registries. You can also link your own private AWS S3 bucket to Buildkite Packages, which allows you to:
+
+- Manage Buildkite registry packages, container images and modules stored within your private AWS S3 bucket (that is, your _private storage_). Private storage:
+    * Located closer to your geographical location may provide faster registry access.
+    * Mitigates network transmission costs.
+
+- Use Buildkite Packages' management and metadata-handling features to manage these files in registries within your private storage.
+
+- Maintain control, ownership and sovereignty over the packages, container images and modules stored within your Buildkite Packages registries.
+
+Buildkite Packages uses [AWS CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/Welcome.html) to provision its services within your private AWS S3 storage.
+
+## Before you start
+
+Before you can start linking your private AWS S3 storage to Buildkite Packages, you will need to have set yourself up with this storage.
```

Comment:
"set yourself up with this storage" might be a little ambiguous. Perhaps:

Before you can start linking your private AWS S3 storage to Buildkite Packages, you will need to have created a bucket.

Reply:
Yeah, during our team content crit session yesterday, @mbelton-buildkite called this out as not being that clear too. I clearly rushed this change and committed and merged this branch in before reviewing it myself.

I've fixed this in #2830 .